tests/integration/test_dictionaries_dependency/test.py

```diff
@@ -61,11 +61,6 @@ def check():
 
     check()
 
-    # Restart must not break anything.
-    node.restart_clickhouse()
```

Comment:
@tavplubix I am not sure why this tests before succeded, probably because of some luck or dictionaries attach ordering. Actually if dictionary is point to other dictionary, we cannot rely on their order of initialization and with lazy_load this test will fail.